data/korean/bbc_korean_radio/liveradio.json

```diff
@@ -43,7 +48,7 @@
     ]
   },
   "promo": {
-    "name": "BBC News 코리아 라디오",
+    "name": "BBC 코리아 라디오",
```

Comment:
Is `News` meant to be removed here? This will affect the page metadata/SEO

Reply:
It came from cell F25 here: https://docs.google.com/spreadsheets/d/1YopqX8FFtoA1tffcU8PtrZUCOZvI96miKO-b88BhofY/edit#gid=0

Think it's intentional - cc @aeroplanejane

Reply:
Yes, content on the spreadsheet has been reviewed by editorial and Tom Chandler and matches the headline that was previously on the radio page.